tun/tun_linux.go

```diff
@@ -380,6 +380,9 @@ func (tun *NativeTun) Close() error {
 func CreateTUN(name string, mtu int) (Device, error) {
 	nfd, err := unix.Open(cloneDevicePath, os.O_RDWR, 0)
 	if err != nil {
+		if os.IsNotExist(err) {
+			return nil, fmt.Errorf("CreateTUN(%q) failed; tuntap %s does not exist", name, cloneDevicePath)
```

Comment:
On linux, tun and tap are separate. WireGuard only uses `tun`, since it's layer 3.